SQLQuery_3(homework).sql

```diff
@@ -0,0 +1,303 @@
+CREATE DATABASE ZooManagement;
+
+USE ZooManagement;
+
+CREATE TABLE ZooInstitutions (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Name VARCHAR(100) NOT NULL, 
+    City VARCHAR(150) DEFAULT 'Country Romania',
+);
+
+
+CREATE TABLE Visitors (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfDay VARCHAR(50) NOT NULL,
+    FirstName VARCHAR(100) NOT NULL, 
+    LastName VARCHAR(100) NOT NULL,
+    ZooId INT REFERENCES ZooInstitutions(Id)
+
+);
+
+
+CREATE TABLE Tickets (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfTicket VARCHAR (100) NOT NULL,
+    Price MONEY CHECK(Price > 1) NOT NULL DEFAULT 1,
+    StartDate DATE CHECK(StartDate >= GetDate()),
+    EndDate DATE CHECK(EndDate >= GetDate()+1),
+    VisitorId INT REFERENCES Visitors(Id),
+    ZooId INT REFERENCES ZooInstitutions(Id)
+);
+
+CREATE TABLE Animals (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    TypeOfSpecies VARCHAR (100) NOT NULL UNIQUE,
+    NumberOfSpecies INT DEFAULT 0,
+    ZooId INT REFERENCES ZooInstitutions(Id)
+);
+
+CREATE TABLE Caretakers (
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    FirstName VARCHAR(100) NOT NULL, 
+    LastName VARCHAR(100) NOT NULL,
+    Salary MONEY CHECK(Salary > 100) DEFAULT 100,
+    ZooId INT REFERENCES ZooInstitutions(Id)
+);
+
+CREATE TABLE Caretakers_Animals(
+    Id INT IDENTITY(1,1) PRIMARY KEY,
+    Caretaker INT REFERENCES Caretakers(Id),
```

Comment:
The same naming convention with CaretakerId and AnimalId should be used. By being consistent we create easy to understand and easy work with databases.